catalog/app/utils/usePreviousSync.spec.js

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react'
+import { act, renderHook } from '@testing-library/react-hooks'
+
+import usePreviousSync from './usePreviousSync'
+
+describe('utils/usePreviousSync', () => {
+  it('should return the same value', () => {
```

Comment:
fyi the point of the original hook was to return the previous value, not the same one